Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs

```diff
@@ -34,5 +34,6 @@ public enum ExecutionStrategyType
         MySqlPrepareDatabaseAndRunQueries = 16,
         MySqlRunQueriesAndCheckDatabase = 17,
         MySqlRunSkeletonRunQueriesAndCheckDatabase = 18,
+        RubyExecutionStrategy = 32
```

Comment:
Strategies are now ordered by language in groups (makes it more readable and easier to find what you're looking for). Ruby currently "stands out" as it is the only Ruby related strategy, but still conforms to the aforementioned idea.